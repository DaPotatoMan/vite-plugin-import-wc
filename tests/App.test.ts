import { it, expect, describe } from 'vitest'
import getApp from './App.vue?ce'


describe('plugin',async () => {
  const App = await getApp('my-app')
  const app = new App()

  it('can mount to element', async () => {
    const body = document.createElement('body')
    body.appendChild(app)
    expect(body).toMatchSnapshot()
  })
})
