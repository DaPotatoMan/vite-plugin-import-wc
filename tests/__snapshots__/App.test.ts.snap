// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`plugin > can mount to element 1`] = `
<body>
  <my-app />
</body>
`;
